FITist_project/user_profiles/templates/user_profiles/program.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'layout.html' %}
+
+{% block title %}Workout of the Day{% endblock %}
+
+{% block content %}
+    <p>Program Name: {{ program.name }}</p>
+    <p>Number Of Weeks: {{ program.total_weeks }}</p>
+    {% for workout in workouts %}
```

Comment:
It might be nice to see this workout stuff broken up into a more specific (possibly reusable) template that you just include here.